### PR TITLE
Ceph dash

### PR DESCRIPTION
**Pull request by fernandosanchezmunoz**
A tiny little dashboard to see how you're DC/OS Ceph cluster is doing today.

Tutorial for this one will be merged immediately in the main Ceph tutorial as a section. Authentication is common to both packages and... hey, who would run a Ceph cluster and NOT want to see how it's doing?

### CODE REVIEW

**Comment by fernandosanchezmunoz** [comment]
can you please merge this and the new Ceph so that I can do the example over the merged packages instead of handling two different branches?

File: repo/packages/C/ceph-dash/0/resource.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "icon-large": "https://s3.amazonaws.com/downloads.mesosphere.io/universe/assets/icon-service-ceph-large.png",
+    "screenshots": [
+      "https://raw.githubusercontent.com/crapworks/ceph-dash/master/screenshots/ceph-dash.png",
+      "https://raw.githubusercontent.com/crapworks/ceph-dash/master/screenshots/ceph-dash-graphite.png"
+    ]
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "ceph-dash-docker": "fernandosanchez/ceph-dash:0.1"
```
Comment (ryadav88) [inline repo/packages/C/ceph-dash/0/resource.json]: should be pulled from `mesosphere` org so we can get usage of packages created by Mesosphere

**Comment by ryadav88** [comment]
> can you please merge this and the new Ceph so that I can do the example over the merged packages instead of handling two different branches?

Ok for this one but the point of asking for examples is that the reviewer can test the package based on the `example`. This will change when we have tests in place.

File: repo/packages/C/ceph-dash/0/resource.json (hunk 1 above)
Comment (fernandosanchezmunoz) [inline repo/packages/C/ceph-dash/0/resource.json]: happy to host it at the mesosphere account, let's work to see how we can get that done.

**Pull request merged**